### ISSUE TITLE
Two issues with project using custom.sty

### ISSUE THREAD

**Issue by sewhite** [sillsdev/ptx2pdf#455]
This project uses a custom.sty to specify a font for Hebrew characters in the acrostic headings in Ps 119. I have said to use custom.sty in the Advanced page. 
There are two issues with PTXPrint.
1) When it runs, it says \s cannot occur at the beginning of Ps 119, but only under \c \esb \periph
But custom.sty does specify it can occur under \qa
![image](https://user-images.githubusercontent.com/16904461/119835600-f94c4e00-bece-11eb-9574-b9fc22fc6db3.png)
2) The desired font for Hebrew characters is only used twice, in the first two dividers. In the others, the box character shows the given character is not found in the font. The font, SBL Hebrew, is installed for all users.
(The markup is inconsistent, that the first two acrostic headings use \qa, while the later ones use \s, but the custom style is permitted to occur under both \qa and \s so this shouldn't be an issue. 

[MP11-DefaultPTXprintArchive.zip](https://github.com/sillsdev/ptx2pdf/files/6554415/MP11-DefaultPTXprintArchive.zip)

**Comment by davidg-sil** [comment]
I think the 2nd problem is that your custom marker (`\heb`, I assume) isn't getting used as much as you think.
```
\c 119
\qa Alep \heb א \heb*
\s Wapômbêŋ anêŋ abô balabuŋ
```
but:
```
\q3 anêm abô ami.
\s ג Gimel
\s Anyô hapôm malaiŋ ma hadahaliŋ abô balabuŋ
```

(this is in the 'original' file as included in the archive, so should be before any canonicalisation, etc. has taken place, as far as I understand)

**Comment by sewhite** [comment]
Thanks for that point, David. So I consider the second point as resolved (user error).

**Comment by markpenny** [comment]
@sewhite Could you please check to see if the 1st point has been resolved in 1.9
There were some interesting bugs squashed last week, and I'm hoping this also solves your issue. But if not, then let us know and we'll go around again...
